powergenome/load_construction.py

```diff
@@ -0,0 +1,711 @@
+#######################
+# Header.R
+
+from datetime import time
+from operator import index
+from os import path, times
+import numpy as np
+import pandas as pd
+import os
+import logging
+
+from pathlib import Path
+
+from pandas.core.reshape.merge import merge
+from powergenome.util import regions_to_keep
+from powergenome.us_state_abbrev import state2abbr, abbr2state
+
+path_in = r"..\data\load_profiles_data\input"  # fix
```

Comment:
Will need to set this somewhere outside the script. Either in a standard folder under `\data` or possibly specified by user in the .env file.